postprocessing/janus-pp-rec.c

```diff
@@ -801,8 +802,29 @@ int main(int argc, char *argv[])
 			/* Simple enough... */
 			started = TRUE;
 			highest_rtp_ts = rtp_ts;
+			highest_seq = p->seq;
 			p->ts = (times_resetted*max32)+rtp_ts;
 		} else {
+			if(!video && !data && rtp->markerbit == 1) {
+				/* Try to detect RTP silence suppression */
+				int32_t seq_distance = abs((int16_t)(p->seq - highest_seq));
+				if(seq_distance < 100) {
```

Comment:
I guess the `100` here could be made configurable (default=100) via command line, in case it's too aggressive (or not aggressive enough) in some specific cases.

Reply:
done